<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1181</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Circuit Diagram Designer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Resources/resistor.png</normaloff>Resources/resistor.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="verticalFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QTreeWidget" name="componentsTree">
         <column>
          <property name="text">
           <string notr="true">Components</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchTextEdit">
         <property name="placeholderText">
          <string>Type to search</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1181</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="menubarAction_New"/>
    <addaction name="separator"/>
    <addaction name="menubarAction_Open"/>
    <addaction name="separator"/>
    <addaction name="menubarAction_Save"/>
    <addaction name="menubarAction_SaveAs"/>
    <addaction name="separator"/>
    <addaction name="menubarAction_ExportAsImage"/>
    <addaction name="menubarAction_Print"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="menubarAction_Help"/>
    <addaction name="menubarAction_About"/>
    <addaction name="menubarAction_AboutQt"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="menubarAction_ZoomIn"/>
    <addaction name="menubarAction_ZoomOut"/>
   </widget>
   <addaction name="menuMain"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menubarAction_New">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="menubarAction_Open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="menubarAction_Save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="menubarAction_SaveAs">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="menubarAction_Help">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="menubarAction_About">
   <property name="text">
    <string>About program</string>
   </property>
  </action>
  <action name="menubarAction_AboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="menubarAction_ExportAsImage">
   <property name="text">
    <string>Export as image</string>
   </property>
  </action>
  <action name="menubarAction_Print">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="menubarAction_ZoomIn">
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="menubarAction_ZoomOut">
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
